src/styles/index.scss

```diff
@@ -1,3 +1,114 @@
+@import 'https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap';
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  color: #060B35;
 }
+
+.card {
+  width: 198px;
+  padding: 31px 0 16px;
+  border: 1px solid hsl(0deg, 0%, 95%);
+  border-radius: 5px;
+
+  &__container {
+    padding: 0 16px;
+  }
+
+  &__image {
+    margin: 0 auto;
+    box-sizing: border-box;
+    background-image: url(../images/imac.jpeg);
+    background-position: center;
+    background-size: contain;
```

Comment:
```suggestion
    background-size: cover;
```

Reply:
if i add border boxing to all elements, then the whole card gets broken. its necessary to add?
and i have problems with width of the image and something wrong with the button, could you help me?